test/service/test_request_validation.py

```diff
@@ -265,3 +271,17 @@ def test(url, message, params):
             test(url,
                  params={},
                  message='Missing required query parameters `entity_type`, `integration_type`')
+
+    def test_bad_catalog_param(self):
+        url = furl(url=self.base_url, path='/index/files').url
+        for invalid_catalog_name, error_message in [
```

Comment:
```suggestion
        for catalog, error in [
```

The smaller the scope, the shorter the variable names. Look for other places where you can apply this.